src/activities/ActivityUsageEntity.js

```diff
@@ -457,6 +457,21 @@ export class ActivityUsageEntity extends Entity {
 		return scoreOutOfEntity ? scoreOutOfEntity.properties.gradeType : undefined;
 	}
 
+	/**
+	 * @returns {string} Grade type title for the numeric grade type.
+	 */
+	numericGradeTypeTitle() {
+		const action = this._getScoreOutOfAction();
+		if (action && action.hasFieldByName('gradeType')) {
+			const numericGradeType = action.getFieldByName('gradeType').value.find(x => x.value === 'Numeric');
+			if (numericGradeType) {
+				return numericGradeType.title;
+			}
+		}
+		const scoreOutOfEntity = this._getScoreOutOfEntity();
```

Comment:
No it's not needed, just thought it might be better than returning undefined. I agree that returning a non-numeric term would be confusing when coming from a function called `numericGradeTypeTitle`. Maybe an empty string would be better.